forge-gui/res/cardsfolder/upcoming/grey_knight_paragon.txt

```diff
@@ -0,0 +1,9 @@
+Name:Grey Knight Paragon
+ManaCost:4 W
+Types:Creature Astartes Knight
+PT:4/4
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDestroy | TriggerDescription$ Rites of Banishment — When CARDNAME enters the battlefield, destroy target attacking creature. If that creature is a Demon, exile it instead.
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | SubAbility$ DBExile | SpellDescription$ Destroy target attacking creature. Draw a card.
```

Comment:
Also needs Condition to not destroy when exiling

Reply:
This wording probably doesn’t mean replacement effect

Reply:
Any pointers to similar cards I should look at ?

Reply:
I think you want to use Branch effect – Grisly Sigil might help

If you can't sort it just ping me, I love Branch effect 😁

Reply:
Thanks with the brach effect it's working